crates/kit/src/to_disk.rs

```diff
@@ -247,15 +247,34 @@ impl ToDiskOpts {
                 tty=--tty
             fi
 
+            # Workaround for issue #126: Copy image from additionalimagestore to local storage
+            # without signatures, since bootc install requires changing layer representation.
+            export STORAGE_OPTS=additionalimagestore=${AIS}
+            SOURCE_REF={SOURCE_IMGREF}
+            LOCAL_IMGREF="containers-storage:bcvk-temp-install:latest"
```

Comment:
It's going to be quite a UX hit to make a temporary deep copy of the input image just for this. I already feel the speed hit from having one copy, going to two would be painful.

I'm not opposed to a workaround but I think we should at least spend some time investigating better fixes - which would probably be in https://github.com/containers/container-libs/

See especially the conversation in https://github.com/containers/image/pull/2590

I think we can reopen a related issue from that discussion or so?

---

In the short term maybe we can detect if the image has attached signatures and only do the fallback copy then?